From b4251d599bfec1c5cbd4134d88267a12e0398813 Mon Sep 17 00:00:00 2001
From: Michael W <baddaemon87@gmail.com>
Date: Sun, 24 Jun 2018 14:02:14 +0200
Subject: [PATCH 16/28] vendor: Fix install[boot|recovery]

* Newer adb (server version 39) doesn't have state "online" anymore
* Use "device" instead
* Fixes: "adb: unknown state online; expected 'any', 'bootloader',
  'device', 'recovery', or 'sideload'"

Change-Id: I3a7a4d418d276c9648f61a4a378d2857c545a070
---
 build/envsetup.sh | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/build/envsetup.sh b/build/envsetup.sh
index a2b668a2..3b18fd5e 100644
--- a/build/envsetup.sh
+++ b/build/envsetup.sh
@@ -368,11 +368,11 @@ function installboot()
         fi
     fi
     adb start-server
-    adb wait-for-online
+    adb wait-for-device
     adb root
     sleep 1
-    adb wait-for-online shell mount /system 2>&1 > /dev/null
-    adb wait-for-online remount
+    adb wait-for-device shell mount /system 2>&1 > /dev/null
+    adb wait-for-device remount
     if (adb shell getprop ro.lineage.device | grep -q "$LINEAGE_BUILD");
     then
         adb push $OUT/boot.img /cache/
@@ -417,11 +417,11 @@ function installrecovery()
         fi
     fi
     adb start-server
-    adb wait-for-online
+    adb wait-for-device
     adb root
     sleep 1
-    adb wait-for-online shell mount /system 2>&1 >> /dev/null
-    adb wait-for-online remount
+    adb wait-for-device shell mount /system 2>&1 >> /dev/null
+    adb wait-for-device remount
     if (adb shell getprop ro.lineage.device | grep -q "$LINEAGE_BUILD");
     then
         adb push $OUT/recovery.img /cache/
-- 
2.17.1

